src/Diagnostics.RuntimeHost/Controllers/ObserverController.cs

```diff
@@ -18,12 +18,27 @@ public class ObserverController : Controller
         [HttpGet(UriElements.ObserverGetSites)]
         public async Task<IActionResult> GetSiteDetails(string siteName)
         {
+            if (string.IsNullOrWhiteSpace(siteName))
```

Comment:
Ignore this file for changes as this code path is only executed when using LocalHostObserver which were not using in PublicAzure and we'll soon migrate from in national clouds.